runtime/crab/src/migrations.rs

```diff
@@ -23,6 +27,19 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> Weight {
-	0
-	// RuntimeBlockWeights::get().max_block
+	Scheduler::migrate_v2_to_v3();
+
+	for precompile in CrabPrecompiles::<Runtime>::used_addresses() {
+		EVM::create_account(&precompile, vec![0x60, 0x00, 0x60, 0x00, 0xFD]);
+	}
+
+	let module = b"FromDarwiniaIssuing";
+	migration::remove_storage_prefix(module, b"MappingFactoryAddress", &[]);
+	migration::remove_storage_prefix(module, b"RemoteBackingAccount", &[]);
+
+	// TODO
+	// Do we need a migration for KtonTreasury?
```

Comment:
Any insight about this? @AurevoirXavier The other changes LGTM

Reply:
in progress